IotcRestApi/regulator_draft.md

```diff
@@ -0,0 +1,145 @@
+            "traceId":string
+        }
+    ]
+}
+```
+
+# Zařízení a podporované endpointy
+
+## Teplotní regulátor
+
+### POST api/temperature-regulator/{DeviceId}/mode
+
+Změna módu teplotního regulátoru.
+
+Předávané parametry:
+
+| Parametr    | Typ         | Povinný | Popis                               |
+|:------------|:------------|:--------|:------------------------------------|
+| requestId   | string-guid | ano     | Jednoznačný identifikátor requestu. |
+| mode        | string      | ano     | Cílový mód zařízení.                |
+
+Cílový mód zařízení může nabývat těchto hodnot:
+
+| Hodnota            | Název                       |
+|--------------------|-----------------------------|
+| basic-regulator    | Základní regulace teploty.  |
+| manual             | Manuální regulace.          |
+| summer             | Letní režim.                |
+
+Ukázka requestu:
+
+```yaml
+{
+    "requestId": "b5e5a8e4-d09d-4d0f-8878-5ab24c2647fc",
+    "mode": "basic-regulator"
+}
+```
+
+### GET api/temperature-regulator/{DeviceId}/mode
+
+Zjištění módu teplotního regulátoru.
+
+Ukázka response:
+
+```yaml
+{
+    "mode": "basic-regulator"
+}
+```
+
+### POST api/temperature-regulator/{DeviceId}/temperature
+
+Nastavení cílové teploty pro regulaci.
+Podporováno pouze, pokud mód regulátoru je `basic-regulator`.
+
+Předávané parametry:
+
+| Parametr           | Typ         | Povinný | Popis                               |
+|:-------------------|:------------|:--------|:------------------------------------|
+| requestId          | string-guid | ano     | Jednoznačný identifikátor requestu. |
+| target-temperature | float       | ano     | Cílová teplota.                     |
+
+Ukázka requestu:
+
+```yaml
+{
+    "requestId": "b5e5a8e4-d09d-4d0f-8878-5ab24c2647fc",
+    "target-temperature": 21.5
+}
+```
+
+### GET api/temperature-regulator/{DeviceId}/temperature
+
+Zjištění cílové teploty pro regulaci.
+Podporováno pouze, pokud mód regulátoru je `basic-regulator`.
+
+Ukázka response:
+
+```yaml
+{
+    "target-temperature": 21.5
+}
+```
+
+### POST api/temperature-regulator/{DeviceId}/position
+
+Nastavení cílové polohy termohlavic.
+Podporováno pouze, pokud mód regulátoru je `manual`.
+
+Předávané parametry:
+
+| Parametr           | Typ         | Povinný | Popis                               |
+|:-------------------|:------------|:--------|:------------------------------------|
+| requestId          | string-guid | ano     | Jednoznačný identifikátor requestu. |
+| target-temperature | float       | ano     | Cílová teplota.                     |
+
+Ukázka requestu:
+
+```yaml
+{
+    "requestId": "b5e5a8e4-d09d-4d0f-8878-5ab24c2647fc",
+    "position": 50
+}
+```
+
+### GET api/temperature-regulator/{DeviceId}/position
+
+Zjištění cílové polohy termohlavic.
+Podporováno pouze, pokud mód regulátoru je `manual`.
+
+Ukázka response:
+
+```yaml
+{
+    "position": 50
+}
+```
+
+### POST api/temperature-regulator/{DeviceId}/factory-reset
```

Comment:
Hadam ze to taky bude PUT a ze to bude mit v body requestId.

Reply:
ok